src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -214,26 +220,27 @@ public JObject ValidateAddress(string address)
             var res = new JObject();
             res["isvalid"] = false;
 
-            // P2PKH
-            if(BitcoinPubKeyAddress.IsValid(address, ref this.Network))
+            // P2WPKH
+            if (BitcoinWitPubKeyAddress.IsValid(address, ref this.Network))
```

Comment:
why did you change the order of these?

Reply:
I was unable to test otherwise. The tests for P2W passed on if statements before they got to P2W validations.

Reply:
Yes the ones that don't check Witness keys and scripts fail on the witness script and key with that error. I'm not sure what else I could do about it in this case. Ofcourse I could always revert the ordering change and disable the tests.